_posts/2015-03-26-webcomponents-0.6.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "webcomponents.js 0.6.0 release"
+date:   2015-03-26 15:32:00
+categories: updates
+author: polymer-team
+excerpt: The 0.6.0 version of the webcomponent.js polyfills have been released - the first major release since the polyfills moved to their new home at [webcomponents.org](https://www.webcomponents.org).
+
+---
+
+Exciting news - the 0.6.0 version of the `webcomponents.js` polyfills has been [released](https://github.com/webcomponents/webcomponentsjs/tree/v0.6.0). A lot of critical fixes are in this release - highlights are described below.
+
+This also marks about 6 months since the polyfills were moved from the Polymer organization over to WebComponents.org. Here are some stats comparing the Polymer/platform repository (the precursor to webcomponents.js) activity in the 6 months prior to the move, to the WebComponents/webcomponentsjs repository in the six months since its creation:
+
+|                         | Before: Polymer/platform (3/29/2014-10/17/2014) | After:  WebComponents/webcomponentsjs (10/5/2014-3/21/2015) |
+|-------------------------|-----------------------------------------|-----------------------------------------------------|
+| Contributors            | 3                                       | 26                                                  |
+| Commits                 | 64                                      | 155                                                 |
+| Non-Polymer Team Contributors | 0                                       | 19                                                  |
+| Non-Polymer Team Commits     | 0                                       | 50                                                  |
+
+A few things you'll note:
+- Huge increase in activity overall.
+- Large chunk of commits are now coming from outside the Polymer team.
+- The Polymer team's activity on the Polyfills has picked up significantly as well.
+
+
+All of these things are great news for the future of the polyfills and the continued growth of Web Components - we couldn't be more excited.
+
+On to the highlights of 0.6.0:
+
+## 0.6.0 Highlights
+
+* Relative path resolving in CSS is fixed. Previously if a stylesheet referenced a file with a relative path, the path would not be resolved correctly.  A major shout-out to [nazar-pc](https://github.com/nazar-pc), who created a PR that fixes this.
```

Comment:
In some cases fixes we've landed spread across a single PR. Where that isn't the case though, it may be useful to link up to these PRs. An alternative is that we link up to a full changelog and folks can go look at what changed manually.